src/ansys/mechanical/core/embedding/app.py

```diff
@@ -239,7 +239,7 @@ def execute_script(self, script: str) -> typing.Any:
         if script_result.Error is not None:
             error_msg += f": {script_result.Error.Message}"
             raise Exception(error_msg)
-        return script_result.Value
+        return str(script_result.Value)
```

Comment:
@koubaa This is modified to be consistent with the remote session for future embedded rpc tests integration.

Reply:
@dipinknair I don't agree with this change. The server can do this, the embedding app must not

Reply:
Yeah, that is also true. Okay we can do that on server then.